examples/basic/main.tf

```diff
@@ -1,5 +1,24 @@
 provider "azurerm" {
-  features {
+  features {}
+}
 
-  }
-}
\ No newline at end of file
+
+data "azurerm_management_group" "source" {
```

Comment:
you cannot use your example. An example must be deployable without any customization. Create new management groups for example

Reply:
edited example to make deployment without customization possible